packages/contracts/deploy/new/40_finalize-managing-dao/40_revoke-permissions-on-plugin-repos.ts

```diff
@@ -0,0 +1,60 @@
+import {DAO__factory, PluginRepo__factory} from '../../../typechain';
+import {Operation} from '../../../utils/types';
+import {getContractAddress, managePermissions, Permission} from '../../helpers';
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {ethers} = hre;
+  const [deployer] = await ethers.getSigners();
+
+  // Revoke `ROOT_PERMISSION`, `MAINTAINER_PERMISSION` and `UPGRADE_REPO_PERMISSION` from `Deployer` on the permission manager of each PluginRepo.
+  for (const repoName in hre.aragonPluginRepos) {
```

Comment:
In my removal PR https://github.com/aragon/osx/pull/551, I am removing `hre.aragonPluginRepos` and everything related to plugins completely.
Why do we still need this script?

Reply:
We weren't planning to have your PR merged a week ago. Since I had to branch out from main, it needed fixes like this. Ofc, you're right, most of the things not needed anymore but for this specific case, we follow this path unfortunatelly.